pkgs/applications/editors/jupyter-kernels/octave/kernel.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, octave, python3Packages }:
+
+with python3Packages;
+
+buildPythonPackage rec {
+  pname = "octave_kernel";
+  version = "0.32.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0dfbxfcf3bz4jswnpkibnjwlkgy0y4j563nrhaqxv3nfa65bksif";
+  };
+
+  buildInputs = [ metakernel ipykernel ];
+
+  # Tests require jupyter_kernel_test to run, but it hasn't seen a
+  # release since 2017 and seems slightly abandoned.
+  # Doing fetchPypi on it doesn't work, even though it exists here:
+  # https://pypi.org/project/jupyter_kernel_test/.
```

Comment:
Did you try something like https://github.com/NixOS/nixpkgs/blob/7f651d15e2a370e253f10e21c396ed0b8d52e3b6/pkgs/development/python-modules/geojson/default.nix#L10 ?

Reply:
I tried various arguments, including a few different things for `format` but couldn't make it work.